data/cyp2c9_substrate_carbonmangels/meta.yaml

```diff
@@ -0,0 +1,76 @@
+name: cyp2c9_substrate_carbonmangels
+description: |-
+  CYP P450 2C9 plays a major role in the oxidation of both xenobiotic
+  and endogenous compounds. Substrates are drugs that are metabolized by the enzyme.
+  TDC used a dataset from Carbon Mangels et al, which merged information on substrates
+  and nonsubstrates from six publications.
+targets:
+- id: CYP2C9_Substrate
+  description: The drugs that are metabolized by the CYP P450 2C9 (1) or not (0)
+  units: substrate
+  type: categorical
+  names:
+  - CYP P450 2C9 Substrate
+  - CYP2C9 Substrate
+  - ADME Drug metabolism
+  - Pharmacokinetics metabolism
+  - Substrate toward CYP2C9
+  uris:
+  - https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C26633
```

Comment:
i would really be specific with the URIs and only add them it they really match the target